open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/models/MyPlacesDataTypeInfo.java

```diff
@@ -25,19 +29,33 @@
      * Responsible for launching an editor.
      */
     private final MyPlacesEditListener myTypeController;
+    
 
     /**
      * Constructs a new MyPlacesDataTypeInfo.
      *
-     * @param tb The toolbox.
+     * @param toolbox The toolbox.
      * @param placemark The placemark this data type represents.
      * @param placeTypeController Responsible for launching an editor.
      */
-    public MyPlacesDataTypeInfo(Toolbox tb, Placemark placemark, MyPlacesEditListener placeTypeController)
+    public MyPlacesDataTypeInfo(Toolbox toolbox, Placemark placemark, MyPlacesEditListener placeTypeController)
     {
-        super(tb, Constants.MY_PLACES_LABEL, placemark.getId(), placemark.getName(), placemark.getName(), false);
+        super(toolbox, Constants.MY_PLACES_LABEL, placemark.getId(), placemark.getName(), placemark.getName(), false);
         myKmlPlacemark = placemark;
         myTypeController = placeTypeController;
+        
+        initialize();
+    }
+
+    private void initialize()
+    {
+        OrderParticipantKey orderKey;

```

Comment:
You could combine this into one statement, e.g. OrderParticipantKey orderKey = new DefaultOrderParticipantKey...

Reply:
true, lemme fix